sql/pgsql/inc/TPgSQLStatement.h

```diff
@@ -26,6 +28,12 @@ struct PgSQL_Stmt_t {
    PGresult *fRes;
 };
 
+#else
+
+struct PgSQL_Stmt_t {};
```

Comment:
Is it intentional to remove the `PGresult *fRes;`struct  member?

Reply:
It is just for dictionary generation. 
While member is not persistent - we can put anything here to make cling happy.

Reply:
OK, I see. Thanks